libcloudforensics/providers/aws/forensics.py

```diff
@@ -206,9 +206,25 @@ def StartAnalysisVm(
   Returns:
     Tuple[AWSInstance, bool]: a tuple with a virtual machine object
         and a boolean indicating if the virtual machine was created or not.
+
+  Raises:
+    RuntimeError: When multiple AMI images are returned.
   """
+
   aws_account = account.AWSAccount(
       default_availability_zone, aws_profile=dst_profile)
+
+  # If no AMI ID is given we use the default Ubuntu 18.04
+  # in the region requested.
+  if not ami:
+    qfilter = [{'Name':'name', 'Values':[UBUNTU_1804_FILTER]}]
+    ami_list = aws_account.ListImages(qfilter)
+    # We should only get 1 AMI image back, if we get multiple we
+    # have no way of knowing which one to use.
+    if len(ami_list) > 1:
+      raise RuntimeError('error - ListImages returns >1 AMI image!')
```

Comment:
It would be nice to display the names of the images that were returned so that the user can quickly adjust their filter.

Reply:
Done